Shared/Realm.Shared/Handles/QueryHandle.cs

```diff
@@ -226,38 +226,38 @@ public void BinaryNotEqual(IntPtr columnIndex, IntPtr buffer, IntPtr bufferLengt
             nativeException.ThrowIfNecessary();
         }
 
-        public void StringContains(IntPtr columnIndex, string value)
+        public void StringContains(IntPtr columnIndex, string value, bool caseSensitive = true)
```

Comment:
I wanted to have a default parameter as a hint toward what's the default behavior that we want (e.g. with `==`). Otherwise, every time the user has not explicitly specified case sensitivity, we'll have to figure out what to default to.